src/cms/views/pages/page_view.py

```diff
@@ -275,25 +262,28 @@ def post(self, request, *args, **kwargs):
         )
 
         published = page_translation.status == status.PUBLIC
-        if not page_instance:
-            if published:
-                messages.success(
-                    request, _("Page was successfully created and published")
-                )
-            else:
-                messages.success(request, _("Page was successfully created"))
-        elif not page_translation_instance:
-            if published:
-                messages.success(
-                    request, _("Translation was successfully created and published")
-                )
-            else:
-                messages.success(request, _("Translation was successfully created"))
-        else:
-            if published:
-                messages.success(request, _("Translation was successfully published"))
+        if not unchanged:
```

Comment:
```suggestion
        if not page_form.has_changed() and not page_translation_form.has_changed():
            messages.info(request, _("No changes detected, but date refreshed"))
        else:
```
I think this can be solved without the additional variable.